Header files not being installed

My library built atop jemalloc builds fine with 856db56f6ec54f59491fa7897dab9a23d5bf9ff4 but breaks with 41e0b857bef0b787a581c7a8334b46981d5e06ed:

```
/scratch/users/pakin/sicm-ci-test/tmp/include/jemalloc/jemalloc.h:246:10: fatal error: jemalloc/jemalloc_defs.h: No such file or directory
  246 | #include "jemalloc/jemalloc_defs.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

As the compiler error suggests, `jemalloc_defs.h` is nowhere to be found in the installation directory:

```console
$ find /scratch/users/pakin/sicm-ci-test/tmp/ -type f
/scratch/users/pakin/sicm-ci-test/tmp/include/jemalloc/jemalloc.h
/scratch/users/pakin/sicm-ci-test/tmp/share/doc/jemalloc/jemalloc.html
/scratch/users/pakin/sicm-ci-test/tmp/share/man/man3/jemalloc.3
/scratch/users/pakin/sicm-ci-test/tmp/lib/libjemalloc.so.2
/scratch/users/pakin/sicm-ci-test/tmp/lib/libjemalloc.a
/scratch/users/pakin/sicm-ci-test/tmp/lib/pkgconfig/jemalloc.pc
/scratch/users/pakin/sicm-ci-test/tmp/lib/libjemalloc_pic.a
/scratch/users/pakin/sicm-ci-test/tmp/bin/jeprof
/scratch/users/pakin/sicm-ci-test/tmp/bin/jemalloc.sh
/scratch/users/pakin/sicm-ci-test/tmp/bin/jemalloc-config
```

--- a/include/jemalloc/jemalloc_protos.h.in
+++ b/include/jemalloc/jemalloc_protos.h.in
@@ -1,6 +1,3 @@
-#include "jemalloc/jemalloc_defs.h"
-#include "jemalloc/jemalloc_macros.h"
-
 /*
  * The @je_@ prefix on the following public symbol declarations is an artifact
  * of namespace management, and should be omitted in application code unless

